specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2023-09-01/restorePoint.json

```diff
@@ -713,24 +689,32 @@
           "items": {
             "$ref": "#/definitions/RestorePointSourceVMDataDisk"
           },
-          "x-ms-identifiers": [
-            "lun"
-          ],
+          "x-ms-identifiers": ["lun"],
           "description": "Gets the data disks of the VM captured at the time of the restore point creation."
+        },
+        "diskControllerType": {
+          "$ref": "#/definitions/RestorePointSourceVMDiskControllerType",
+          "description": "Gets the disk controller type of the VM captured at the time of the restore point creation."
         }
       },
       "description": "Describes the storage profile."
     },
+    "RestorePointSourceVMDiskControllerType": {
+      "type": "string",
+      "enum": ["SCSI", "NVMe"],
+      "x-ms-enum": {
+        "name": "DiskControllerTypes",
+        "modelAsString": true
+      },
+      "description": "Specifies the disk controller type configured for the VM and VirtualMachineScaleSet. This property is only supported for virtual machines whose operating system disk and VM sku supports Generation 2 (https://docs.microsoft.com/en-us/azure/virtual-machines/generation-2), please check the HyperVGenerations capability returned as part of VM sku capabilities in the response of Microsoft.Compute SKUs api for the region contains V2 (https://docs.microsoft.com/rest/api/compute/resourceskus/list). For more information about Disk Controller Types supported please refer to https://aka.ms/azure-diskcontrollertypes."
```

Comment:
Provides the disk controller type configured for the VM. This property is only supported for virtual machines whose operating system disk and VM sku supports Generation 2 (https://docs.microsoft.com/en-us/azure/virtual-machines/generation-2). For more information about Disk Controller Types supported please refer to https://aka.ms/azure-diskcontrollertypes.